### ISSUE TITLE
Sequence / Order of todo items not preserved and no way of moving items up and down

### ISSUE THREAD

**Issue by cruftex** [tmaegel/ntodotxt#74]
I am using todo.txt with a text editor and the CLI tools. An app to quickly capture "buy milk @grocer" would be really great.

In my daily usage, in the todo.txt file the top most items are the ones that are clear and actionable and most relevant at that moment. However, this order is not preserved in the app and there is also no way rearrange the order of the items.

**Comment by jaschop-1k5o** [comment]
I agree with the UX observation that the default alphabetical sorting is not that helpful. However I would caution against bringing the todo.txt ordering into it. It seems hacky to assign that much meaning to line orders, when todo.txt offers other mechanisms, e.g. a default sort by creation date (also see #38 ).

Reordering operations also seem like an invitation for sync conflicts, possibly duplicating tasks.

**Comment by jaschop-1k5o** [comment]
@flako-dd I referenced reordering since it was in the issue title, but I don't have an issue at all with your comment.

While free reordering doesn't sound practical with todo.txt, line order could well be the default display order unless a sort is explicitly selected. (Order should then probably be reverted, with the last line becoming the first item.)
